App.tsx

```diff
@@ -11,16 +11,12 @@ import { Camera, useCameraDevices } from 'react-native-vision-camera'
 import Reanimated, { useAnimatedStyle, useSharedValue } from 'react-native-reanimated'
 import { readFile, unlink } from 'react-native-fs'
 import Config from 'react-native-config'
-import { Buffer } from 'buffer'
+import axios from 'axios'
 import { format } from 'date-fns'
 import { Rekognition } from 'aws-sdk'
 
-const rekognition = new Rekognition({
-  region: 'ap-northeast-1',
-  credentials: {
-    accessKeyId: Config.AWS_ACCESS_KEY_ID,
-    secretAccessKey: Config.AWS_SECRET_ACCESS_KEY,
-  }
+const client = axios.create({
+  baseURL: Config.API_ENDPOINT,
```

Comment:
> Keep in mind this module doesn't obfuscate or encrypt secrets for packaging, so do not store sensitive keys in .env. It's [basically impossible to prevent users from reverse engineering mobile app secrets](https://rammic.github.io/2015/07/28/hiding-secrets-in-android-apps/)[](https://github.com/luggit/react-native-config#setup), so design your app (and APIs) with that in mind.
https://github.com/luggit/react-native-config